models/stg_stripe.yml

```diff
@@ -486,6 +486,11 @@ models:
             - invoice_line_item_id
             - invoice_id
             - source_relation
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - unique_invoice_line_item_id
+            - invoice_id
+            - source_relation
```

Comment:
Is this necessary? Wouldn't it make more sense to include `unique_invoice_line_item_id` in the unique combination of columns test above as opposed to creating a new test?

Reply:
Actually, do we even need to include this in the test? It seems the original unique combination of columns test is passing just fine. Since the unique_id is an artifact of an older Stripe API, should we just remove the test on that field altogether? It seems like testing on invoice_id and invoice_line_id is sufficient. What are your thoughts?

Reply:
Yeah I would not create a new test in this release. I would prefer we remove the failing one and then either not test the `unique_id` anymore or include it in the existing test. 

This may be a hot take, but I would actually recommend we don't include the `unique_id` at all and we just remove the test. From your investigation we have found that the `unique_id` is an artifact from a release from January 2020. We also can see the unique combo test on invoice_id and invoice_line_id is working as expected. I would worry that adding the `unique_id` to that test would not actually be an accurate representation of the uniqueness of the table.

Reply:
As per internal discussion with team, will remove the uniqueness test from unique_id entirely. Will see if any changes are needed on the connector end.